classes/teacher.rb

```diff
@@ -1,9 +1,8 @@
 require_relative 'person'
-# teacher subclass
 
 class Teacher < Person
-  def initialize(specialization, _id, _age, _name = 'unkown', parent_permission: true)
-    super
+  def initialize(specialization)
+    super()
```

Comment:
- [ ] Please fix the error when creating a person.

![image](https://user-images.githubusercontent.com/38635158/170824447-a1a2df62-c3cf-407c-a9d5-4ad2a66e1ceb.png)

_Next code reviewer please check if creating rentals or list the rentals. Right now ı can't create any person so ı can't create the rentals._